packages/lib/services/e2ee/EncryptionService.ts

```diff
@@ -72,10 +75,11 @@ export default class EncryptionService {
 	//
 	// So making the block 10 times smaller make it 100 times faster! So for now using 5KB. This can be
 	// changed easily since the chunk size is incorporated into the encrypted data.
-	private chunkSize_ = 5000;
+	public chunkSize_ = 65536;
 	private decryptedMasterKeys_: Record<string, DecryptedMasterKey> = {};
-	public defaultEncryptionMethod_ = EncryptionMethod.SJCL1a; // public because used in tests
-	private defaultMasterKeyEncryptionMethod_ = EncryptionMethod.SJCL4;
+	public defaultEncryptionMethod_ = EncryptionMethod.StringV1; // public because used in tests
+	public defaultFileEncryptionMethod_ = EncryptionMethod.FileV1; // public because used in tests
```

Comment:
Any reason why we need a separate method for files?

Reply:
Here are the steps for encrypt a file:
1. The blob libraries read the file content then return to the JS side as a base64-encoded string (the first encode)
2. The crypto library convert the string into bytes before encryption.
3. The crypto library takes the binary input and generate binary ciphertext.
4. The binary ciphertext is converted to base64-encoded string and saved to file. (the second encode)
The `FileV1` method makes the Step 2 as a decoding step, which cancels out the size overhead from Step 1, so the encrypted will have smaller compared with the old implementation. We could use the `StringV1` method, but that would make the ciphertext of file encryption larger.
As for string encryption, we don't have Step 1 and the string input is not base64 encoded, so we cannot use the same method.

Reply:
I don't understand what this means:

> The FileV1 method makes the Step 2 as a decoding step, which cancels out the size overhead from Step 1,

But if FileV1 is more efficient, why not use the same method for StringV1?